Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM docker-upgrade.artifactory.build.upgrade.com/container-base:20180917-9
+LABEL maintainer=devops@upgrade.com
+
+USER root
+
+ENV I_KEY i_key
+ENV S_KEY s_key
+ENV HOST host
+ENV COLL_ENDPOINT endpoint
+ENV SCAN_INTERVAL_IN_SEC scan_interval_in_sec
+
+RUN curl  https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm --output /tmp/epel7.rpm && rpm -Uvh /tmp/epel7.rpm && yum install epel-release
+RUN yum install -y python36 python36-libs python36-devel python36-setuptools && python3.6 -m ensurepip
+RUN pip3 install six botocore
+
+COPY lambda_function.py /opt/sumologic-duo-security/
+COPY duo_client /opt/sumologic-duo-security/duo_client
```

Comment:
Instead of the 2 `COPY` instructions, you could just do a single `ADD` instuction to add the local folder (`ADD . /opt/sumologic-duo-security/`, for example)

Reply:
So, I don't want to add Dockerfile and jenkinsfile. so, it means I don't want to copy all localfile

Reply:
thanks. added